executor/benchmark_test.go

```diff
@@ -844,6 +853,56 @@ func BenchmarkBuildHashTableForList(b *testing.B) {
 	}
 }
 
+// TestingAggMemTracker is used for testing the memory tracker of aggregation executor,
+// because it uses functionality provided in this package, it is placed here
+func TestingAggMemTracker(c *check.C) {
```

Comment:
How about using this test function `TestMemoryUsageAfterClose` to test it?

Reply:
Yes, that's a good idea, I was waiting for your selection PR to be merged, now it is already done so I'll remove this part and use your test function.